src/main/java/games/diamant/actions/OutOfCave.java

```diff
@@ -3,12 +3,14 @@
 import core.AbstractGameState;
 import core.actions.AbstractAction;
 import core.interfaces.IPrintable;
+import games.diamant.DiamantGameState;
 
 // This action is used when the player is out of the Cave
 public class OutOfCave extends AbstractAction implements IPrintable {
```

Comment:
Proposed refactor for Diamant: there are many action classes that do nothing (or do the exact same thing). Should be just one class, maybe with an enum parameter for the different options.

Reply:
Agreed - that's how I'd do it - but it works, is straightforward to understand given how simple the game is, so changing it now feels a little too purist.

Reply:
Actually, on this - I've done this refactor already with the Map Elites work we did a while back, that I still have to get into the main TAG repo. So scrap this comment, it'll be done... eventually.